configs/defaults/seqr_loader.toml

```diff
@@ -63,4 +63,7 @@ target_loci = """AFF2,AR,ARX_1,ARX_2,ATN1,ATXN1,ATXN10,ATXN2,ATXN3,ATXN7,ATXN8OS
 BEAN1,C9ORF72,CACNA1A,CBL,CNBP,COMP,DAB1,DIP2B,DMD,DMPK,FMR1,FOXL2,FXN,GIPC1,GLS,\
 HOXA13_1,HOXA13_2,HOXA13_3,HOXD13,HTT,JPH3,LRP12,MARCHF6,NIPA1,NOP56,NOTCH2NLC,\
 NUTM2B-AS1,PABPN1,PHOX2B,PPP2R2B,PRDM12,RAPGEF2,RFC1,RUNX2,SAMD12,SOX3,STARD7,TBP,\
-TBX1,TCF4,TNRC6A,XYLT1,YEATS2,ZIC2,ZIC3"""
\ No newline at end of file
+TBX1,TCF4,TNRC6A,XYLT1,YEATS2,ZIC2,ZIC3"""
+
+[images]
+stripy = 'australia-southeast1-docker.pkg.dev/cpg-common/images/stripy:v2.2'
```

Comment:
Is the idea of having an `images` section in some of these default configs to override the newest versions that would come from the image config template?

Reply:
That's because we don't manage `stripy` through images because wasn't clearly versioned: https://github.com/populationgenomics/images/pull/50#issuecomment-1310104799
(though I checked again in the gitlab and there is v2.2 tag now, so can add it).

Reply:
Adding stripy: https://github.com/populationgenomics/images/pull/62